include/deal.II/hp/dof_handler.h

```diff
@@ -884,6 +898,39 @@ namespace hp
     virtual std::size_t
     memory_consumption() const;
 
+    /**
+     * Whenever serialization with a parallel::distributed::Triangulation as the
+     * underlying triangulation is considered, we also need to consider storing
+     * the active_fe_indices on all active cells as well.
+     *
+     * This function registers that these indices are to be stored whenever the
+     * parallel::distributed::Triangulation::save() function is called on the
+     * underlying triangulation.
+     *
+     * @note Currently only implemented for triangulations of type
+     *   parallel::distributed::Triangulation.
+     *
+     * @note See parallel::distributed::SolutionTransfer for further information.
```

Comment:
I think you can use `@see` here. You may want to check how that is rendered to know what sentence structure is useful.

(In reality, I think both ways are fine and it's ok if you don't want to make the change. I just wanted to point out the existence of `@see` -- http://www.doxygen.nl/manual/commands.html#cmdsee and the compiled example here: http://www.doxygen.nl/manual/examples/autolink/html/class_autolink___test.html)

Reply:
I'll give it a try and post a screenshot here later.

Reply:
You agree with this?
![doxygen-hp-serialization](https://user-images.githubusercontent.com/18285973/52169075-cea0bf00-2732-11e9-9964-c66e56266443.png)

Reply:
It looks a bit clunky with the grammar. How about write it as this:
```
  @see The documentation of X::Y has further information on serialization.
```